src/scanoss/winnowing.py

```
@@ -111,11 +157,21 @@ class Winnowing(ScanossBase):
     a list of WFP fingerprints with their corresponding line numbers.
     """
 
-    def __init__(self, size_limit: bool = False, debug: bool = False, trace: bool = False, quiet: bool = False,
-                 skip_snippets: bool = False, post_size: int = 32, all_extensions: bool = False,
-                 obfuscate: bool = False, hpsm: bool = False,
-                 strip_hpsm_ids=None, strip_snippet_ids=None, skip_md5_ids=None
-                 ):
+    def __init__(
```

Ruff (0.8.2)

Too many arguments in function definition (12 > 5)

(PLR0913)